packages/blog/preset/_assets/rocket-blog.css

```diff
@@ -23,12 +23,25 @@ body[layout='layout-blog-details'] #sidebar-nav li.anchor a:hover::before {
 }
 
 .articles article {
+  display: flex;
+  flex-direction: column;
+  position: relative;
   flex: 0 1 360px;
   margin: var(--gap) 0 0 var(--gap);
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
   transition: 0.3s;
 }
 
+.articles article a[href]::after {
```

Comment:
this will result in users no longer being able to highlight/copy content

### Current:

https://user-images.githubusercontent.com/24378/111266069-4cca3200-862a-11eb-9ce8-3a1e14ea95d3.mp4

### With Change: 

https://user-images.githubusercontent.com/24378/111265907-15f41c00-862a-11eb-9f6d-3fd347c36ccc.mp4

Reply:
you can solve this with a z-index on the text, but that wont work for the link text

Reply:
I can also implement this with a bit of .js
https://codepen.io/erikkroes/pen/JjRLLjZ